### PR TITLE
Rename "animation-time-range" -> "animation-range" to match the accepted spec

### PR DESCRIPTION
**Pull request by dream-ellie**
**TL;DR**
Rename "animation-time-range" to "animation-range" because this is the latest spec (according to [this](https://github.com/flackr/scroll-timeline/issues/64#issuecomment-1544020128) comment).

**WHY**
At the moment polyfill implementation diverged from the latest spec.
That means we can't produce tutorials or educational materials that show use of the latest (accepted) spec because the polyfill itself is not up-to-date with the latest spec.

This PR brings the polyfill a little bit closer to the current version of the spec.

@bramus @flackr @kevers-google My apologies for poking you like this directly in this PR. I am not sure how else to draw attention to this 😀

### CODE REVIEW

**Comment by Gaubee** [comment]
great work. but the author doesn't seem to be maintaining the project anymore? Maybe we should start another project to merge these PRS
